lib/nerves/distillery.ex

```diff
@@ -0,0 +1,90 @@
+if Code.ensure_loaded?(Mix.Releases.Plugin) do
+  defmodule Nerves.Distillery do
+    use Mix.Releases.Plugin
+
+    alias Mix.Releases.{Release, Shell}
+
+    def before_assembly(release, _opts) do
+      if Nerves.Env.loaded?() do
+        vm_args = Map.get(release.profile, :vm_args) || "rel/vm.args"
+        plugins = order_plugins(release.profile.plugins)
+
+        profile =
+          release.profile
+          |> Map.put(:dev_mode, false)
+          |> Map.put(:include_src, false)
+          |> Map.put(:include_erts, System.get_env("ERL_LIB_DIR"))
+          |> Map.put(:vm_args, vm_args)
+          |> Map.put(:plugins, plugins)
```

Comment:
Might be a good idea to do a single `Map.merge/2` here where all of these defaults are defined in a single map.